integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -9340,6 +9340,15 @@ msgstr ""
 "Diese Seite konnte nicht importiert werden, da sie zu einer anderen Region "
 "gehört ({})."
 
+#~ msgid ""
+#~ "hallo aschaffenburg is your digital companion for the town of "
+#~ "Aschaffenburg. Here you can find local information, advice centres and "
+#~ "services."
+#~ msgstr ""
+#~ "hallo aschaffenburg is your digital companion for the town of "
+#~ "Aschaffenburg. Here you can find local information, advice centres and "
+#~ "services."
```

Comment:
I guess this text also has to be added for integreat then. How do you add a translation again?

Reply:
For the backend UI translations (which the translation file belongs to), see: https://digitalfabrik.github.io/integreat-cms/internationalization.html

However, for the social media tags we can't really use this, since it only supports the backend UI languages English, German and Dutch. For the content languages, we need to store the translations in the database. Since we're currently lacking the appropriate model structure for this, I'm afraid this is out of scope for this PR...
Do you want to open a new issue for the meta description of the root level / region selection (simiar to #402 and #522)?

Reply:
See #2563 .

Reply:
For this text we'd also already have translations in the frontend.